src/types/trait-object.md

```diff
@@ -24,7 +24,7 @@ Trait objects are written as the keyword `dyn` followed by a set of trait
 bounds, but with the following restrictions on the trait bounds.
 
 r[type.trait-object.constraint]
-All traits except the first trait must be auto traits, there may not be more than one
+All traits except one trait must be auto traits, there may not be more than one
```

Comment:
I'm having a little bit of trouble clearly understanding what this is saying, because another interpretation is that it is required to have one non-auto trait. I'm wondering if this could be rewritten to be a little more explicit what it means. Perhaps something like:

```suggestion
There may not be more than one non-auto trait, no more than one
```